docs/modules/casper.rst

```diff
@@ -2215,6 +2215,58 @@ Waits until a `JavaScript alert <https://developer.mozilla.org/en-US/docs/Web/AP
         this.echo("Alert received: " + response.data);
     });
 
+.. _casper_waitforexec:
+
+.. index:: Child Process, Spawn, exec File
+
+``waitForExec()``
+-------------------------------------------------------------------------------
+**Signature:** ``waitForExec(command, parameters[, Function onTimeout, timeout])``
```

Comment:
you forgot "Function then"  :smile_cat: 

waitForExec(command, parameters[,Function then, Function onTimeout, Number timeout])